tests/aws/services/apigateway/test_apigateway_api.py

```diff
@@ -416,6 +416,12 @@ def test_update_rest_api_compression(self, apigw_create_rest_api, snapshot, aws_
         )
         snapshot.match("enable-compression", response)
 
+        # check that listing is not exploding after update, null -> 10
+        try:
+            aws_client.apigateway.get_rest_api(restApiId=api_id)
+        except ClientError:
+            assert False
```

Comment:
You don't need the `try...except` clause here, you can leave the exception be raised and it will make the test fail. 
```suggestion

        aws_client.apigateway.get_rest_api(restApiId=api_id)
```